azext_iot/_help.py

```diff
@@ -1340,6 +1340,52 @@
     short-summary: Delete an enrollment group in an Azure IoT Hub Device Provisioning Service.
 """
 
+helps[
+    "iot dps enrollment-group registration"
+] = """
+    type: group
+    short-summary: Manage device registrations for an enrollment group in an Azure IoT Hub Device
+        Provisioning Service.
+"""
+
+helps[
+    "iot dps enrollment-group registration list"
+] = """
+    type: command
+    short-summary: List device registrations for an enrollment group in an Azure IoT Hub Device
+        Provisioning Service.
+"""
+
+helps[
+    "iot dps enrollment-group registration show"
+] = """
+    type: command
+    short-summary: Get a device registration for an enrollment group in an Azure IoT Hub Device
+        Provisioning Service.
+"""
+
+helps[
+    "iot dps enrollment-group registration delete"
+] = """
+    type: command
+    short-summary: Delete a device registration in an Azure IoT Hub Device Provisioning Service.
```

Comment:
Missing 'enrollment group' verbiage like the other enrollment-group registration commands.

Reply:
done